### PR TITLE
Copy post and comment information to clipboard

### PR DESCRIPTION
**Pull request by Noxor11**
Allows to copy the thumbnail url of a post if available, and copy a comment made to a post.

### CODE REVIEW

**Comment by MV-GH** [comment]
The three composables are almost the change them into a generic something like MyThumbnailCopy and pass the differences as arguments

**Comment by Noxor11** [comment]
Comments don't have thumbnails, but I agree the common code should be contained in a method

File: app/src/main/java/com/jerboa/ui/components/post/PostListing.kt [hunk 1]
```diff
@@ -1443,6 +1445,19 @@ fun PostOptionsDialog(
                         icon = Icons.Outlined.Block,
                         onClick = onBlockCommunityClick,
                     )
+                    if (postView.post.thumbnail_url != null) {
+                        IconAndTextDrawerItem(
```
Comment (twizmwazin) [inline app/src/main/java/com/jerboa/ui/components/post/PostListing.kt:1449]: This should probably be grouped with the other copy options in this menu, also allowing the creator to copy the thumbnail link.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -66,21 +66,25 @@
     <string name="comment_edit_type_your_comment">Type your comment</string>
     <string name="comment_mention_node_block">Block %1$s</string>
     <string name="comment_mention_node_copy_permalink">Copy Permalink</string>
+    <string name="comment_mention_node_copy_comment">Copy Comment</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:69]: Copy comment text

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -66,21 +66,25 @@
...
     <string name="comment_mention_node_go_to">Go to %1$s</string>
     <string name="comment_mention_node_permalink_copied">Permalink Copied</string>
+    <string name="comment_mention_node_comment_copied">Comment Copied</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:72]: Comment text copied

**Review by twizmwazin — CHANGES_REQUESTED** [review]
One other thing, I'm noticing that the dialog is not being dismissed after selecting the added options. If you're feeling up to it, maybe add in the option to copy post text as well while we're here.

**Comment by Noxor11** [comment]
I just realized there's an Utils file, so I used that to place the new function.
Now the popup gets dismissed after clicking, and added ability to copy title, name, and text from a post.

**Review by twizmwazin — APPROVED** [review]

**Pull request merged**